kubernetes/aws-eks/blueprint.yaml

```diff
@@ -368,6 +368,9 @@ node_templates:
     type: cloudify.nodes.aws.ec2.NATGateway
     properties:
       client_config: *client_config
+      resource_config:
+        kwargs:
+          ConnectivityType: private
```

Comment:
@kaplanyaniv Does the cluster work with private connectivity? Did you try deploying something on it, like an application and a service?

Reply:
I did not, I'm not sure how to check that